src/services/entity-types.js

```diff
@@ -68,6 +93,22 @@ module.exports = class EntityTypeHelper {
 		try {
 			bodyData.updated_by = loggedInUserId
 			if (bodyData.value) bodyData.value = bodyData.value.toLowerCase()
+
+			if (bodyData.hasOwnProperty('is_external') && bodyData['is_external'] != '') {
```

Comment:
@adithyadinesh0412  we can optimize 
if ('is_external' in bodyData && bodyData.is_external !== '"") {